build.sbt

```diff
@@ -720,15 +722,17 @@ lazy val docSettings = Seq(
   )
 
 
+/*
 // Documentation is generated for projects defined in
 // `docsSourcesAndProjects`.
 lazy val docs = project
   .enablePlugins(MicrositesPlugin)
+  .enablePlugins(ScalaUnidocPlugin)
+  .enablePlugins(GhpagesPlugin)
   .settings(moduleName := "scalding-docs")
   .settings(sharedSettings)
   .settings(noPublishSettings)
-  .settings(unidocSettings)
-  .settings(ghpages.settings)
   .settings(docSettings)
-  .settings(tutScalacOptions ~= (_.filterNot(Set("-Ywarn-unused-import", "-Ywarn-dead-code"))))
+  .settings(scalacOptions in Tut ~= (_.filterNot(Set("-Ywarn-unused-import", "-Ywarn-dead-code"))))
```

Comment:
side note: we should migrate to mdoc, tut is EOL I believe.

also, I though `Tut / scalacOption` is the new syntax?

Reply:
I'll change it. Copied from `Tut` docs.

Reply:
Commented this line for now,  will migrate to mdoc in separate PR